config/prod/htan-data-model-web-ec2.yaml

```diff
@@ -0,0 +1,50 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-linux-v1.j2
+stack_name: htan-data-model-web-ec2
+sceptre_user_data:
+  Distro: "aws"
+  # (Optional) Expose ports to incoming traffic (default is no open ports) (valid range: 1-65535)
+  OpenPorts: ["8000", "9200"]
```

Comment:
this is ok.  was wondering if this website should only be accessible by sage employees or should it be accessible by people outside of sage as well?

Reply:
The site should be accessible outside Sage as well.
It'd be great to assign a domain name as well eventually, we'll have to decide on the name first w/in HTAN.